src/app/getting-started/services/getting-started.service.ts

```diff
@@ -67,13 +75,9 @@ export class GettingStartedService implements OnDestroy {
   getExtProfile(id: string): Observable<ExtUser> {
     let url = `${this.usersUrl}/${id}`;
     return this.http
-      .get(url, { headers: this.headers })
-      .map(response => {
-        return response.json().data as ExtUser;
-      })
-      .catch((error) => {
-        return this.handleError(error);
-      });
+      .get<ExtUserResponse>(url, { headers: this.headers })
+      .map(response => response.data)
+      .catch((error) => this.handleError(error));
```

Comment:
please consider adding types for `response` and `error`

Reply:
PR opened here: https://github.com/fabric8-ui/fabric8-ui/pull/3263